### PR TITLE
function for checking current availability of package

### PR DESCRIPTION
**Pull request by KPostOffice**
This checks whether a specific python package version has gone missing from an index.  This is required for implementing a sieve to not include missing packages in adviser

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -3933,6 +3933,25 @@ def update_missing_flag_package_version(
                 .update({"is_missing": value}, synchronize_session='fetch')
             )
 
+    def is_python_package_version_is_missing(
+        self,
+        package_name: str,
+        package_version: str,
+        index_url: str,
+    ) -> bool:
+        """Check whether is missing flag is set for python package version."""
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:3942]: `is_missing`

**Comment by harshad16** [comment]
recheck

**Pull request merged**